apps/services/user-profile/src/app/user-profile/userToken.controller.ts

```diff
@@ -52,9 +58,11 @@ export class UserTokenController {
       nationalId,
     )
     if (!userProfile) {
-      throw new NotFoundException(
+      this.logger.info(
```

Comment:
probably missing this here
import { LOGGER_PROVIDER, type Logger } from '@island.is/logging'

Reply:
Fixed :heart: